# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/gcr3/hotfix-shared.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- gcr-3.41.2/meson_post_install.py.vanilla	2025-10-15 16:00:33.814348617 +0200
+++ gcr-3.41.2/meson_post_install.py	2025-10-15 16:01:37.204414735 +0200
@@ -21,15 +21,6 @@
 if os.environ.get('DESTDIR'):
   sys.exit(0)
 
-print('Update icon cache...')
-subprocess.call(['gtk-update-icon-cache', '-f', '-t', icondir])
-
-print('Compiling gsettings schemas...')
-subprocess.call(['glib-compile-schemas', schemadir])
-
-print('Updating MIME database...')
-subprocess.call(['update-mime-database', mimedatabasedir])
-
 # FIXME: after a major version bump, just drop this
 print('Creating symlink for libgcr-{}.so'.format(gcr_major_version))
 
@@ -47,5 +38,3 @@
 
     if not os.path.exists(_get_path_for_lib(libgcr_ui_basename)):
         continue
-
-    subprocess.call(['ln', '-f', '-s', libgcr_ui_basename, _get_path_for_lib(libgcr_basename)])
